### PR TITLE
initial commit of rGEDI installation

### PR DESCRIPTION
**Pull request by nmt28**
notebook for installing R v4.1.0 and rGEDI - including relevant dependancies

### CODE REVIEW

**Comment by nmt28** [comment]
@wildintellect

**Comment by wildintellect** [comment]
@nmt28 
1. should we call the example folder `rGEDI` in anticipation of adding some more examples once the install is working?
2. Do you want to keep this in a branch until it's working? then we can merge to `main`?

File: rGEDI/rGEDI-installation.ipynb [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "technological-maryland",
+   "metadata": {},
+   "source": [
+    "#### conda install -y -c conda-forge mamba\n",
+    "#### mamba create -n r4-base -c conda-forge r-base=4.1.0 r-irkernel\n",
+    "#### mamba install -c conda-forge r-untis r-terra r-gtable r-colorspace r-lattice r-vctrs r-viridisLite r-RColorBrewer r-munsell r-labeling r-farver r-Matrix r-nlme r-scales r-mgcv r-MASS r-isoband r-RcppEigen r-deldir r-interp r-jpeg r-png r-wk r-proxy r-s2 r-DBI r-KernSmooth r-class r-e1071 r-units r-lwgeom r-classInt r-sf r-abind r-BH r-data.table r-stringi r-gridExtra r-ggplot2 r-terra r-leaflet.providers r-viridis r-sp r-raster r-markdown r-bit r-zoo r-hexbin r-latticeExtra r-misc3d r-RcppArmadillo r-stars r-rlas r-rgl r-leaflet r-bit64 r-rasterVis r-plot3D r-lidR r-leafsync r-hdf5r r-getPass\n",
+    "\n",
+    "#### run apt install libsz2 in terminal"
```
Comment (chuckwondo) [inline rGEDI/rGEDI-installation.ipynb:12]:
```suggestion
    "```\n",
    "conda install -y -c conda-forge mamba\n",
    "mamba create -n r4-base -c conda-forge r-base=4.1.0 r-irkernel\n",
    "mamba install -c conda-forge r-untis r-terra r-gtable r-colorspace r-lattice r-vctrs r-viridisLite r-RColorBrewer r-munsell r-labeling r-farver r-Matrix r-nlme r-scales r-mgcv r-MASS r-isoband r-RcppEigen r-deldir r-interp r-jpeg r-png r-wk r-proxy r-s2 r-DBI r-KernSmooth r-class r-e1071 r-units r-lwgeom r-classInt r-sf r-abind r-BH r-data.table r-stringi r-gridExtra r-ggplot2 r-terra r-leaflet.providers r-viridis r-sp r-raster r-markdown r-bit r-zoo r-hexbin r-latticeExtra r-misc3d r-RcppArmadillo r-stars r-rlas r-rgl r-leaflet r-bit64 r-rasterVis r-plot3D r-lidR r-leafsync r-hdf5r r-getPass\n",
    "apt install libsz2\n",
    "```"
```

**Comment by wildintellect** [comment]
https://github.com/MAAP-Project/Community/issues/597

**Pull request closed without merging**